server/src/mob/index.js

```diff
@@ -0,0 +1,32 @@
+import { Events } from '../events';
+
+export const MOBS_TYPE = ['B', 'T', 'R'];
+export const MOBS_FOR_ONE_PLAYER = { 'B': 100, 'T': 10, 'R': 10 };
+
+export class Mob {
+  constructor(id, game, type, spawnPoint) {
+    this.id = id;
+    this.game = game;
+    this.type = type;
+    this.alerted = false;
+    this.life = [3, 3];
+    this.x = spawnPoint[0];
+    this.y = spawnPoint[1];
+    this.vx = 0;
+    this.vy = 0;
+  }
+
+  onMove({ x, y, vx, vy }) {
```

Comment:
we need this when we get a position from one player, to broadcast it to other players. the server will move the mobs, we only need to broadcast the positions